README.md

```diff
@@ -82,6 +82,17 @@ This provider enables you to generate one Storybook instance per opened PR of a
 
 You must set `GITHUB_TOKEN` environment variable to access the GitHub.com API.
 
+### With Gitlab (opened MRs)
+
+This provider enables you to generate one Storybook instance per opened MR in a Gitlab repository.
+
+| Key        | Default | Description                                             |
+|------------|---------|---------------------------------------------------------|
+| baseUrl    | -       | **(mandatory)** baseUrl of gitlab deploy                |
```

Comment:
I like the idea of having the possibility to choose a custom a base URL, in order to support self-hosted or business instances. But I prefer to have it as an **optional** option (and to set `gitlab.com` as the default base URL).